<?xml version='1.0' encoding='UTF-8'?>
<testsuite hostname="local.local" name="org.tests.TestCase" tests="4"
           errors="0" failures="1" skipped="1" time="88.386" timestamp="2019-08-07T10:12:43">

    <testcase classname="org.tests.TestCase"
              name="This test PASSED in 2.067 seconds"
              time="2.067">

    </testcase>
    <testcase classname="org.tests.TestCase"
              name="is created in 1 zone with 1 node of 1024 capacity each then indexing and searching of data works as expected"
              time="6.946">

    </testcase>
    <testcase classname="org.tests.TestCase"
              name="This test FAILED in 4.974 seconds"
              time="4.974">
        <failure message="1 is not 2"
                 type="org.scalatest.exceptions.TestFailedException">
            Body text
            Probably stacktrace
            ...
            ...
            ...
        </failure>
    </testcase>
    <testcase classname="org.tests.TestCase"
              name="This test was SKIPPED in 1.115 seconds"
              time="1.115">
        <skipped/>
    </testcase>

    <system-out><![CDATA[]]></system-out>
    <system-err><![CDATA[]]></system-err>
</testsuite>
